README.md

```diff
@@ -1,38 +1,51 @@
 # roc-eln-docker
 
-This project installs the necessary services (dockers) to run a full electronic laboratory book. A deployed version can be found at www.c6h6.org.
+This project installs the necessary services (dockers) to run a full electronic laboratory book. A deployed version that can be used with a Google Account can be found at [www.c6h6.org](http://www.c6h6.org).
 
-## Install docker and docker-compose and some dependencies
+For common questions, please have a look at the [FAQ](faq.md). For some background information about the ELN [you can have a look at our website](https://cheminfo.github.io/eln.epfl.ch/).
+
+## Installing dependencies
+
+### 1. Install docker and docker-compose and some dependencies
+
+On Red Hat Linux/CentOS you need to run the following commands:
 
 ```bash
 yum update -y
 yum install epel-release -y
 yum install git docker docker-compose -y
 ```
 
-## Install Node.js
+You can use [similar commands](https://docs.docker.com/compose/install/) on other operating systems.
+
+### 2. Install Node.js
+
+On Red Hat Linux/CentOS you need to run the following commands:
 
 ```bash
 curl -sL https://rpm.nodesource.com/setup_10.x | bash -
```

Comment:
is node actually needed? Also node 10 will soon no longer be maintained

Reply:
thanks for giving it a look, i updated the readme to a newer node version. Several script actually use node. We'll look into creating an ansible role (will be here https://github.com/kjappelbaum/ansible-role-cheminfo-eln until we transfer ownership to cheminfo) that should make the setup easier.